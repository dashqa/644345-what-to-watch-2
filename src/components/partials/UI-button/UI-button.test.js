import React from 'react';
import renderer from 'react-test-renderer';
import UIButton from './UI-button';

const mock = {
  classMods: ``,
  title: ``
};

it(`UI button correctly renders after relaunch`, () => {
  const {classMods, title} = mock;
  const tree = renderer.create(
      <UIButton
        classMods={classMods}
        title={title}
        onClick={()=>{}}
      />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
