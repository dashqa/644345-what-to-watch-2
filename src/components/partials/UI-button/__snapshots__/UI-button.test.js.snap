// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI button correctly renders after relaunch 1`] = `
<button
  className="btn movie-card__button"
  onClick={[Function]}
  type="button"
>
  
  <span>
    
  </span>
</button>
`;
